docs/area-owners.md

```diff
@@ -74,15 +74,15 @@ Note: Editing this file doesn't update the mapping used by the `@msftbot` issue
-| area-System.Diagnostics.PerformanceCounter     | @jeffhandley  | @adamsitnik @carlossanlop @jozkee                   |                                                                                                                                                                                                                                                                                           |
+| area-System.Diagnostics.EventLog               | @ericstj      | @Anipik @ViktorHofer                                |                                                                                                                                                                                                                                                                                           |
+| area-System.Diagnostics.PerformanceCounter     | @ericstj      | @Anipik @ViktorHofer                                |                                                                                                                                                                                                                                                                                           |
 | area-System.Diagnostics.Process                | @jeffhandley  | @adamsitnik @carlossanlop @jozkee                   |                                                                                                                                                                                                                                                                                           |
-| area-System.Diagnostics.Tracing                | @tommcdon     | @noahfalk @tommcdon @Anipik @ViktorHofer @tarekgh   | Included: <ul><li>System.Diagnostics.DiagnosticSource</li><li>System.Diagnostics.TraceSource</li></ul>                                                     |
-| area-System.Diagnostics.TraceSource            | @jeffhandley  | @buyaa-n @joperezr @krwq                            | Consultants: @tquerec @josephisenhour                                                                                                                                                                                                                                                     |
+| area-System.Diagnostics.Tracing                | @tommcdon     | @noahfalk @tommcdon @Anipik @ViktorHofer @tarekgh   | Included: <ul><li>System.Diagnostics.DiagnosticSource</li><li>System.Diagnostics.TraceSource</li></ul>                                                                                                                                                                                    |
```

Comment:
```suggestion
| area-System.Diagnostics.Tracing                | @tommcdon     | @noahfalk @tommcdon @tarekgh   | Included: <ul><li>System.Diagnostics.DiagnosticSource</li></ul>                                                                                                                                                                                    |
```
Make sure to remove TraceSource, you have a new area for that below.  I still think it's a little odd that we don't have a dedicated label for DiagnosticSource given the amount of development happening there.